content.md

```diff
@@ -95,23 +95,23 @@
 
 ## 3. Registering a Member
 
-### 3.1 Start from map view seen in 2.6. Select menu-icon on the top-left of screen
+### 3.1 Start from map view(2.6.) and tap menu-icon at top-left of screen
 ![screenshot](./images/map.jpg)
 <div class="pagebreak"></div>
 
-### 3.2 Select Member Registration from the dropdown menu
+### 3.2 Select *"Member Registration"* from dropdown menu
```

Comment:
No need for the double quotes when the text is highlighted with italics. (apply everywhere)